nengo/synapses.py

```diff
@@ -72,8 +73,13 @@ def general_step(signal, output, x, y, num, den):
         y.appendleft(np.array(output))
 
     def make_step(self, dt, output, method='zoh'):
-        num, den, _ = cont2discrete((self.num, self.den), dt, method=method)
-        num = num.flatten()
+        num, den = self.num, self.den
+        if self.analog:
+            num, den, _ = cont2discrete((num, den), dt, method=method)
+            num = num.flatten()
+
+        if den[0] != 1.:
+            raise ValueError("First element of the denominator must be 1")
```

Comment:
I would just keep the ValueError, because it's nice and explicit if users have to normalize it themselves. Any filters designed using the Scipy functions should already have this property, anyway.